apps/src/face_detection/filesystem_face_detection.cpp

```diff
@@ -6,111 +6,112 @@
  */
 
 #include "pcl/recognition/face_detection/rf_face_detector_trainer.h"
+#include <boost/filesystem.hpp>
```

Comment:
Use `<>` brackets and separate the diff headers by a blank line (pcl then boost)